torchinductor/decomposition.py

```diff
@@ -17,6 +17,7 @@
 
 decompositions = get_decompositions(
     [
+        aten._adaptive_avg_pool2d,
```

Comment:
Let's leave this using the existing lowering, since it is slightly faster due to masked load.